ocs_ci/deployment/deployment.py

```
@@ -3747,6 +3748,363 @@ def _patch_ramen_csv(self, csv_prefix, namespace, new_image):
+
+        if not hasattr(self, "dr_policy_name"):
+            logger.error("[ramenctl-e2e] dr_policy_name not set, skipping e2e tests")
+            return
+
+        logger.info("[ramenctl-e2e] Starting ramenctl e2e test execution")
+
+        ramenctl_repo_url = constants.RAMENCTL_REPO_URL
+        ramenctl_branch = constants.RAMENCTL_REPO_BRANCH
+
+        ramenctl_clone_dir = tempfile.mkdtemp(prefix="ramenctl-e2e-")
+        logger.info(f"[ramenctl-e2e] Using directory: {ramenctl_clone_dir}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)